content/about.md

```diff
@@ -3,4 +3,72 @@ title: About
+solve open-ended challenges as well as to manage their time and prioritise their
+efforts. This teaches valuable teamwork and time-management skills at a scale
+which participants might not otherwise meet until late in their university or
+working lives.
+
+SourceBots is based at the [University of Southampton][uos]’s [Department of
+Electronics and Computer Science][ecs] and is lead by its students, though is an
+independent organisation. It also benefits from contributors outside the
+University, including teachers and professional engineers.
+
+## Robotics Summer Schools
+
+Each year, SourceBots works with the Smallpiece Trust to hosted an annual
+robotics competition for Year 12 students (age 16–17) at the University of
+Southampton.
+
+This partnership allows the SourceBots volunteers to focus on providing the core
+event: an exciting week-long team challenge in which the participants design,
+build and program an autonomous robot, culminating in an action-packed knockout
+competition.
+
+#### Low Barrier to Entry
+
+One of the core aims of the project is to keep the barrier to entry as low as
+possible. Many of the competitors have little or no prior experience of
+programming or electronics. To cater for this, SourceBots volunteers run a
+number of tutorial sessions throughout the week to introduce participants to the
+fundamentals. Every team also has a dedicated mentor to help them through some
+of the more difficult problems they may encounter.
+
+As the competitors have less than a week to build their robot and prepare for
+the competition, SourceBots provides them with all the parts needed to build a
+robot, as well as access to the fully equipped electronics, computer
+laboratories and workshop facilities at the University’s Department of
+Electronics and Computer Science.
+
+## Longer Robotics Competition
+
+In 2018 SourceBots is hosting a much longer version of its successful summer
+school competitions in collaboration with schools in the south and south-east of
+the UK.
+
+This competition sees teams of sixth-form students facing a similar technical
+challenge to that presented in the summer schools, though with the additional
+challenge of a four-month build period during which they must manage their own
+time and efforts.
+
+While direct mentoring of some of the nearby teams is possible, the distance to
+more distant teams mean that other means of support are needed. Therefore, for
+this competition SourceBots focus is on remotely supporting teams and running
+periodic "Tech Days" which teams are invited to attend to get direct support.
+
+## Get Involved
+
+If you would like to contribute to SourceBots, volunteers in the Southampton
+area are always welcome to assist in mentorship and setting up the arena. There
+are also many ways in which you can contribute online, such as writing software
+or preparing tutorial sessions; email us at <sourcebots@gmail.com> if you’d
```

Comment:
Do we own this email address? If not I suggest we set something up like `hello@sourcebots.co.uk` for things like this

Reply:
Kier suggested this address, so I think we own it. I definitely agree that hello@ or info@ or similar is a much better way to go.

Reply:
I have now setup `hello@`, so I suggest we use that